tests/installation/opensuse_welcome.pm

```diff
@@ -26,4 +26,7 @@ sub test_flags {
     return {milestone => 1};
 }
 
+# 'generic-desktop' already checked in wait_boot_past_bootloader
+sub post_run_hook {}
```

Comment:
I'd be more ok if you add a diag message stating that PRH is being overriden, but no biggie at all

Reply:
hm, would we really care about a diag message of something *not* happening? We use the same approach in other test modules as well. I am a friend of more concise output as also my experience is that log file readers are more likely to be overwhelmed by a wall of text rather than miss content :)